### PR TITLE
Create Submodule

### PR DESCRIPTION
**Pull request by nitro-neal**
Create submodule for test vectors pointing to https://github.com/TBD54566975/web5-spec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,23 @@ Interested in contributing instantly? You can make your updates directly without
 
 [![Button to click and edit code in CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/github/TBD54566975/web5-js/main)
 
+
+## Prerequisites
+
+### Cloning
+This repository uses git submodules. To clone this repo with submodules
+```sh
+git clone --recurse-submodules git@github.com:TBD54566975/tbdex-js.git
```
Comment (diehuxx) [inline README.md:57]:
```suggestion
git clone --recurse-submodules git@github.com:TBD54566975/web5-js.git
```

File: README.md [hunk 2]
```diff
@@ -48,6 +48,23 @@ Interested in contributing instantly? You can make your updates directly without
...
+```
+Or to add submodules after cloning
+```sh
+git submodule update --init
+```
+We recommend this config which will only checkout the files relevant to tbdex-js
+```sh
+git -C web5-spec sparse-checkout set hosted
```
Comment (diehuxx) [inline README.md:65]:
```suggestion
git -C web5-spec sparse-checkout set test-vectors
```

File: README.md [hunk 3]
```diff
@@ -48,6 +48,23 @@ Interested in contributing instantly? You can make your updates directly without
...
+### Cloning
+This repository uses git submodules. To clone this repo with submodules
+```sh
+git clone --recurse-submodules git@github.com:TBD54566975/tbdex-js.git
+```
+Or to add submodules after cloning
+```sh
+git submodule update --init
+```
+We recommend this config which will only checkout the files relevant to tbdex-js
```
Comment (diehuxx) [inline README.md:63]:
```suggestion
We recommend this config which will only checkout the files relevant to web5-js
```

**Review by diehuxx — APPROVED** [review]
A couple typos in the README. otherwise lgtm

**Review by frankhinek — APPROVED** [review]

**Pull request merged**